### PR TITLE
update example on luigi-core-api.md navigate()

### PR DESCRIPTION
**Pull request by ryanhs**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Comment by JohannesDoberer** [comment]
Hello Ryan,

thank you for your PR.
I have one question:
Did you open this PR to fix the documentation example or do you want to show how to open a modal with the `navigate()` function.
Docu is wrong and I would change this line in: 
`Luigi.navigation().navigate('/settings', true) // preserve view`.
Could you change this?

If you want to open a modal, we recommend to use the corresponding function `openAsModal`.

Best regards

Johannes

**Comment by ryanhs** [comment]
@JohannesDoberer  Ah yes, i see those awesome functions openAsModal, openAsDrawer, openAsSplitView.

yes the PR is to change the docu of navigate based on the parameters order, it seems docu not following latest update. :)

ah oke, openAsModal use linkManager.navigate() under the hood, but provide higher level function

File: docs/luigi-core-api.md [hunk 1]
```diff
@@ -496,7 +496,7 @@ Navigates to the given path in the application. It contains either a full absolu
 ```javascript
 Luigi.navigation().navigate('/overview')
 Luigi.navigation().navigate('users/groups/stakeholders')
-Luigi.navigation().navigate('/settings', null, true) // preserve view
+Luigi.navigation().navigate('/settings', null, { title: "Settings Menu", size: "m" }) // openAsModal
```
Comment (JohannesDoberer) [inline docs/luigi-core-api.md:499]: We recommend to use the openAsModal function and not the navigate function.
The `preserve view` example is wrong.
Thanks for finding this 👍 !
Would you change the PR for the preserved view fix
`Luigi.navigation().navigate('/settings', true) // preserve view`

Comment (ryanhs) [reply docs/luigi-core-api.md]: Oke @JohannesDoberer  :+1:  thank you

**Review by JohannesDoberer — APPROVED** [review]

**Pull request merged**